server/src/main/java/io/spine/server/bus/Bus.java

```diff
@@ -99,4 +204,127 @@ public void unregister(D dispatcher) {
      * dispatchers of the bus.
      */
     protected abstract DispatcherRegistry<C, D> createRegistry();
+
+    /**
+     * Filters the given messages.
+     *
+     * <p>The implementations may apply some specific validation to the given messages.
```

Comment:
`<p>Each message goes through the filter chain, specific to the {@code Bus} implementation.`